datalad/interface/tests/test_utils.py

```diff
@@ -255,6 +258,10 @@ def test_eval_results_plus_build_doc():
     assert_in("Parameters", doc1)
     assert_in("It's a number", doc1)
 
+    # docstring shows correct overide values of defaults in eval_params
```

Comment:
```suggestion
    # docstring shows correct override values of defaults in eval_params
```
I will add this typo fix and merge/push directly.

Thanks @andycon !